src/components/StaffList/StaffList.scss

```diff
@@ -0,0 +1,8 @@
+.react-autosuggest__input {
+  width: 100%;
+}
+
+
+.react-autosuggest__suggestions-container--open {
+  border: 1px solid #aaa;
+}
```

Comment:
nit: should have a new line at the end of the file

Reply:
👍